test/d2l-fetch-auth/d2l-fetch-auth.test.js

```diff
@@ -132,8 +134,8 @@ describe('d2l-fetch-auth', () => {
 			return auth(getRelativeGETRequest())
 				.then((req) => {
 					expect(req.method).to.equal('GET');
-					expect(req.headers.get('authorization')).to.not.be.defined;
-					expect(req.headers.get('x-csrf-token')).to.not.be.defined;
+					expect(req.headers.get('authorization')).to.be.null;
+					expect(req.headers.get('x-csrf-token')).to.be.null;
```

Comment:
Same issue here -- and `headers.get` returns `null` not `undefined` anyway. A bit more detail about this weird loophole:
https://github.com/chaijs/chai/issues/955